index.html

```diff
@@ -69,7 +69,7 @@
 		<div class="container">
 			<div class="vertical-container">
 				<!-- Slide 1 -->
-				<section id="slide1" class="slide slide1">
+				<section id="slide1" class="slide slide1" style="background-image: url('images/background7.jpg'); background-size:cover;">

```

Comment:
I added my images to my slides: background image here and section images to the others